styles.css

```diff
@@ -327,6 +386,20 @@ h3 {
   text-align: center;
 }
 
+.description-popup {
+  border: 2px solid green;
+  border-radius: 16px;
+  margin: 20px;
+  padding: 16px 18px;
+  z-index: -1;
+}
+
+.description-popup.scale {
+  transition: all 0.4s ease-in-out;
+  z-index: -2;
+  transform: scale(1.05);
```

Comment:
- [OPTIONAL] I suggest making your whole body scrolling instead of the popup only to make sure the design looks good and is similar to the Figma design.

Your design:
![Screenshot from 2023-02-01 18-11-32](https://user-images.githubusercontent.com/97834160/216105285-4f39a349-d108-492c-ba95-8bac70068243.png)



Figma:
![Screenshot from 2023-02-01 18-14-12](https://user-images.githubusercontent.com/97834160/216105333-c1786f97-e3b8-4d51-9aaf-3c77fc0f390b.png)

Reply:
@Omar-Muhamad I will do that surely. Thank you.